Ask about save before starting the compilation

smaximov opened the pull request:
Closes #10.

smaximov committed:
Ask about save before starting the compilation

kwrooijen commented:
Seems to work properly. Maybe add `compilation-ask-about-save` to the README so people know you can modify that? I'll merge when you fix the conflict. Thanks for all your help on the project :)

smaximov committed:
Merge branch 'master' into feat-ask-about-save

smaximov committed:
Mention `compilation-ask-about-save` in README

smaximov committed:
Fix typo

smaximov commented:
@kwrooijen, you're welcome!

I fixed the merge conflict and added a note about `compilation-ask-about-save` to the README (though you may want to rephrase it).

kwrooijen commented:
Looks good, thanks!
